### ISSUE TITLE
'High Contrast White' theme Avatar (TileIcon Background) color

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1030]
**Describe the bug**
TileIcon Background component in the Avatar component has the same text and background-color: black

> What version of the Fundamental Library Styles are you using?
v0.10.0-rc.1

**To Reproduce**
1. Go to https://deploy-preview-1025--fundamental-styles.netlify.app/components/avatar.html
2. Check TileIcon Background component

**Expected behavior**
TileIcon Background component in the Avatar component should have a `color: #fff`

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81855237-ec52fb80-9513-11ea-93e4-135ddfba392f.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - Contrast ratio now 5:1
<img width="377" alt="Screen Shot 2020-10-07 at 5 32 42 PM" src="https://user-images.githubusercontent.com/21978807/95529822-e19c7580-0990-11eb-92cd-1136fb0d03da.png">

**Comment by prsdthkr** [comment]
@Mike-Diaz I just checked this component in the 'High Contrast White' and its not fixed:
![image](https://user-images.githubusercontent.com/43764832/95619008-bf533800-0a22-11eb-9fb3-4d0965b5167b.png)

To reproduce:
* go to https://deploy-preview-1752--fundamental-styles.netlify.app/?path=/docs/components-avatar--icon
* change the theme to 'High Contrast White' from the top right dropdown
* see 'Tile Icon Background' story
